tests/integration/template_complete_run.cfg

```diff
@@ -42,10 +42,13 @@ years = "1850:1854:2",
   vars = "PRECT"
 
   [[ atm_monthly_glb ]]
+  environment_commands = 'module purge; module load gcc/9.2.0-ugetvbp; module load openmpi/4.0.4-hpcx-hghvhj5; export PATH="~ac.zender/bin_chrysalis:${PATH}"; export PATH="~ac.zender/bin_chrysalis:~ac.zender/anaconda/bin:${PATH}"; export LD_LIBRARY_PATH="~ac.zender/lib_chrysalis:~ac.zender/anaconda/lib:${LD_LIBRARY_PATH}"; export NCO_PATH_OVERRIDE=Yes'
```

Comment:
Keeping these `environment_commands` since the new `ncclimo` will be in the next Unified release anyway. So, no need to figure out how to get the latest `ncclimo` from `conda`.